lib/testing_appserver/modules/people/user_creation_form.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require_relative '../../data/user_data'
+require_relative '../../top_toolbar/top_toolbar'
+require_relative 'modules/people_side_bar'
+require_relative 'user_profile'
+
+module TestingAppServer
+  # AppServer People User creation form
+  # https://user-images.githubusercontent.com/40513035/141932021-1800c838-6e02-4ac3-a94a-5c5f5ee2ca32.png
+  class UserCreationForm
+    include PageObject
+    include PeopleSideBar
+    include TopToolbar
+
+    text_field(:first_name, xpath: "//input[@name='firstName']")
+    text_field(:last_name, xpath: "//input[@name='lastName']")
+    text_field(:email, xpath: "//input[@name='email']")
+    label(:activation_link, xpath: "(//input[@name='passwordType']/../../label)[1]")
+    label(:temporary_password, xpath: "(//input[@name='passwordType']/../../label)[2]")
+    div(:generate_password, xpath: "(//div[@class = 'password-field-wrapper']/div)[2]")
+    text_field(:password, xpath: "//input[@name='password']")
+
+    button(:save_form, xpath: "//button[@class='sc-kNPwMy iHxaXx']") # add_id
+
+    div(:success_toast, xpath: "//div[contains(@class, 'Toastify__toast--success')]")
+
+    def initialize(instance)
+      super(instance.webdriver.driver)
+      @instance = instance
+      wait_to_load
+    end
+
+    def wait_to_load
+      @instance.webdriver.wait_until { first_name_element.present? }
+    end
+
+    def create_new_user(params = {}, generate_pwd = false)
+      fill_user_form(params, generate_pwd)
+      @instance.webdriver.move_to_element_by_locator(save_form_element.selector[:xpath])
+      save_form_element.click
+      @instance.webdriver.wait_until { success_toast_element.present? }
+      UserProfile.new(@instance)
+    end
+
+    def fill_user_form(params = {}, generate_pwd = false)
```

Comment:
I think both those method can be private until we found any reason to make them public